engine/standalone/util/src/main/scala/pl/touk/nussknacker/engine/standalone/utils/JsonSchemaStandaloneSourceFactory.scala

```diff
@@ -0,0 +1,72 @@
+package pl.touk.nussknacker.engine.standalone.utils
+
+import com.typesafe.scalalogging.LazyLogging
+import io.circe.Json
+import org.everit.json.schema.Schema
+import org.everit.json.schema.loader.SchemaLoader
+import org.json.JSONObject
+import pl.touk.nussknacker.engine.api.process.SourceTestSupport
+import pl.touk.nussknacker.engine.api.test.{NewLineSplittedTestDataParser, TestDataParser}
+import pl.touk.nussknacker.engine.api.typed.{TypedMap, _}
+import pl.touk.nussknacker.engine.api.{CirceUtil, MethodToInvoke, ParamName}
+import pl.touk.nussknacker.engine.standalone.api.types.GenericResultType
+import pl.touk.nussknacker.engine.standalone.api.{ResponseEncoder, StandalonePostSource, StandaloneSourceFactory}
+import pl.touk.nussknacker.engine.standalone.utils.typed.{SchemaTypingResult, TypedMapUtils}
+import pl.touk.nussknacker.engine.util.json.BestEffortJsonEncoder
+
+import java.nio.charset.StandardCharsets
+
+class JsonSchemaStandaloneSourceFactory extends StandaloneSourceFactory[TypedMap] {
```

Comment:
Shouldn't it override openApiDefinition?

Reply:
Yes it should! Thanks. Updated and I added a test for that.